cddl_test/static/prelude.rs

```diff
@@ -6,9 +6,10 @@ use wasm_bindgen::prelude::*;
 //       instead have it either:
 //  1) generate 1 per each wrapped type
 //  2) don't pass it, instead wrap it/unwrap it automatically
+#[derive(Clone)]
```

Comment:
This was meant for the previous commit related to the RustType refactor to support tags.